tests/test_siaf_interface.py

```diff
@@ -0,0 +1,27 @@
+"""Test the functions provided by siaf_interface.
+
+Authors
```

Comment:
FWIW there is no need to list authors explicitly like this. `git blame` and `git log` will tell you anyway. And such lists will get out of date unless all the authors are careful about updating it. To give proper git/GitHub credits, it is better to amend the commit that added this file to the original author (if possible).

Reply:
I think it's useful to have, if only to be able to quickly know who to send any questions to.